src/main/java/games/strategy/engine/lobby/server/db/BadWordController.java

```diff
@@ -6,27 +6,19 @@
 import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.logging.Logger;
 
 /**
  * Utilitiy class to create/read/delete bad words (there is no update).
  */
 public class BadWordController {
-  private static final Logger logger = Logger.getLogger(BadWordController.class.getName());
-
   public void addBadWord(final String word) {
     try (final Connection con = Database.getPostgresConnection();
-        final PreparedStatement ps = con.prepareStatement("insert into bad_words (word) values (?)")) {
+        final PreparedStatement ps =
+            con.prepareStatement("insert into bad_words (word) values (?) on conflict do nothing")) {
```

Comment:
Currently if a bad word already exists, postgres or any other DB would throw an exception with code 30000 (not sure if the error codes are even universal), this is bad practise since we can just use built in features of postgres.
If a word is already present, we don't need to do anything.

Reply:
Noting the 'if' statement in the exception below, this is a good update to get rid of that. We do lose the context of why the conflict would be okay, would you mind adding a quick comment?

NB: I think this entire class is dead code. I suspect we will use SQL directly to do these inserts and not do it via Java code.